deps/src/main/java/com/microsoft/azure/sdk/iot/deps/ws/impl/WebSocketUpgrade.java

```diff
@@ -43,7 +42,7 @@ public WebSocketUpgrade(String hostName, String webSocketPath, int webSocketPort
      *
      * @param host The host header field value.
      */
-    public final void setHost(String host)
+    private void setHost(String host)
```

Comment:
If we're going through the trouble of making them private. I'm more in favor to make all of these simple mutators Lombok'd with a private access level.

Reply:
Good point. Now is as good a time as any